bashmarks.sh

```diff
@@ -1,169 +1,175 @@
-# Copyright (c) 2010, Huy Nguyen, http://www.huyng.com
-# All rights reserved.
-# 
-# Redistribution and use in source and binary forms, with or without modification, are permitted provided 
-# that the following conditions are met:
-# 
-#     * Redistributions of source code must retain the above copyright notice, this list of conditions 
-#       and the following disclaimer.
-#     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
-#       following disclaimer in the documentation and/or other materials provided with the distribution.
-#     * Neither the name of Huy Nguyen nor the names of contributors
-#       may be used to endorse or promote products derived from this software without 
-#       specific prior written permission.
-#       
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED 
-# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
-# PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR 
-# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
-# TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) 
-# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
-# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
-# POSSIBILITY OF SUCH DAMAGE.
-
-
-# USAGE: 
-# s bookmarkname - saves the curr dir as bookmarkname
-# g bookmarkname - jumps to the that bookmark
-# g b[TAB] - tab completion is available
-# p bookmarkname - prints the bookmark
-# p b[TAB] - tab completion is available
-# d bookmarkname - deletes the bookmark
-# d [TAB] - tab completion is available
-# l - list all bookmarks
-
-# setup file to store bookmarks
-if [ ! -n "$SDIRS" ]; then
-    SDIRS=~/.sdirs
-fi
-touch $SDIRS
-
-RED="0;31m"
-GREEN="0;33m"
-
-# save current directory to bookmarks
-function s {
-    check_help $1
-    _bookmark_name_valid "$@"
-    if [ -z "$exit_message" ]; then
-        _purge_line "$SDIRS" "export DIR_$1="
-        CURDIR=$(echo $PWD| sed "s#^$HOME#\$HOME#g")
-        echo "export DIR_$1=\"$CURDIR\"" >> $SDIRS
-    fi
+#!/bin/bash
+# Copyright (c) 2015, Leonardo Gutiérrez R
+# leogutierrezramirez@gmail.com
```

Comment:
I'm not a project admin for bashmarks, but I doubt that Huy would appreciate his name being replaced and the license info being removed...